source/fundamentals/crud/run-command.txt

```diff
@@ -0,0 +1,167 @@
+.. _golang-run-command:
+
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to run a **database command** with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+Compose a Command
+-----------------
+
+The {+driver-short+} provides the following methods to run database
+commands: 
```

Comment:
A couple items related to this section:

1) The title is "compose a command", but I wasn't able to find information about what it means to compose a command.  
2) The list below shows methods that you can use to wrap the commands, but lacks explanation of what it means to run and compose a database command.
3) Is there any info you can provide to explain when a user might care about using this command? The example shows listCollections for which you can usually use [ListCollections()](https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.10.3/mongo#Database.ListCollections), so it's not self-evident how these wrapper methods are helpful.